lib/resty/acme/autossl.lua

```diff
@@ -216,6 +223,16 @@ function AUTOSSL.update_cert(data)
     AUTOSSL.client_initialized = true
   end
 
+
+  -- If its failed in the past and its still cooling down
+  -- we dont do anything right now
+  local failure_lock_key = certificate_failure_lock_key_prefix .. ":" .. data.domain
```

Comment:
since the prefix already ends with a `:`, we don't actually need an extra `:` here